joomla/joomla-cms
# [plg_search_tags] Search results now don't include unpublished, archived or trashed tags

Pull Request for Issue #10712 .
#### Summary of Changes

Select only published tags from the database by adding `$query->where('a.published = 1');.`
#### Testing Instructions
1. Make sure Search - Tags plugin is enabled
2. Create a tag
3. Unpublish, archive or trash it
4. Type a tag name in a mod_search module
5. Only published tags should be listed.
